### PR TITLE
FEATURE: make summary as first tab on user page

### PR DESCRIPTION
**Pull request by techAPJ**
Requested here: https://meta.discourse.org/t/custom-user-fields-show-on-activity-but-not-summary-page/59196/3?u=techapj

### CODE REVIEW

**Comment by techAPJ** [comment]
Fixed the build.

**Comment by techAPJ** [comment]
@eviltrout can you please review this PR?

File: app/assets/javascripts/discourse/routes/user-activity-index.js.es6 [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-
-  actions: {
-    didTransition() {
-      this._super();
-      this.controllerFor("user").set("indexStream", true);
-      return true;
-    }
-  }
-
+  userActionType: undefined
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/routes/user-activity-index.js.es6]: I prefer `null` when dealing with uninitialized values. `undefined` is good for something that you didn't assign a value, like when you do `let x`.

**Comment by eviltrout** [comment]
I made a trivial comment but otherwise it's good to merge!

File: app/assets/javascripts/discourse/routes/user-activity-index.js.es6 (hunk 1 above)
Comment (techAPJ) [inline app/assets/javascripts/discourse/routes/user-activity-index.js.es6]: Got it, thanks for the tip!

**Comment by techAPJ** [comment]
Thanks @eviltrout!

**Pull request merged**